### ISSUE TITLE
dont load.

### ISSUE THREAD

**Issue by philipehsing** [chrisben/imgcache.js#99]
Hey 

I get this in my console 
 [console.log] WARN: ImgCache not loaded yet! - Have you called ImgCache.init() first?
i used your code in your readme in my ondeviceready.., it works fine in chrome but not in ios/android

Cheers!

**Comment by chrisben** [comment]
Perhaps ImgCache.init failed. Did you see the alert? Or perhaps it failed silently. Turn ImgCache.options.debug on first, then try to add a console.log output just before you call the init method. If you get that warning log entry before your new log, that means somewhere your code calls imgcache methods before the init statement.

**Comment by chrisben** [comment]
@philipehsing Try the new Promises wrapper (QImgCache) within the latest v1.0rc1, this should be easier for you because it makes sure init is called first (and you don't need to call it). Also please check out the new Troubleshooting section in the Readme file.

**Comment by philipehsing** [comment]
Yes i have turned on the debug option. All i get is [console.log] WARN: ImgCache not loaded yet! - Have you called ImgCache.init() first?

What should i do?:( it works on laptopts etc but no phone, where can i find the nre QImgCache?

**Comment by chrisben** [comment]
qimgcache.js can be found in the js/ folder next to imgcache.js within the master branch.

**Comment by philipehsing** [comment]
How do i use qimgcahce?

**Comment by chrisben** [comment]
Read the documentation, if you search for qimgcache, you'll find a link to an example.
